### PR TITLE
Develop Moyo

### PR DESCRIPTION
**Pull request by ShaoLei2**

### CODE REVIEW

**Comment by ShaoLei2** [comment]
- [DEMO LINK](https://ShaoLei2.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ShaoLei2.github.io/layout_moyo-header/report/html_report/)

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,89 @@
+/* stylelint-disable font-family-name-quotes */
+/* stylelint-disable-next-line import-notation */
+/* stylelint-disable-next-line import-notation */
+@import url('https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap');
```
Comment (nazarmatsevych) [inline src/style.css:4]: Looks like this import doesn't work, try to add Roboto font using link tag inside <head>

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,89 @@
...
+
+:root {
+  --main-color: #00acdc;
+}
+
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: 'Roboto','Poppins', sans-serif;
```
Comment (nazarmatsevych) [inline src/style.css:14]: Move font-family and default font styles to html selector

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,89 @@
...
+}
+
+
+.header {
+  box-sizing: border-box;
```
Comment (nazarmatsevych) [inline src/style.css:19]: move box-sizing to * selector

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,89 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap');
+
+:root {
+  --main-color: #00acdc;
+}
+
+
 body {
   margin: 0;
+  padding: 0;
```
Comment (nazarmatsevych) [inline src/style.css:13]: Body doesn't have default padding

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
If you have any questions - feel free to ask them in fe_chat

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Check the font here and consider removing the default body margin

File: src/index.html [hunk 5]
```diff
@@ -5,10 +5,53 @@
...
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Smartphones</a>
+          </li>
+          <li class="nav-item">
+            <a
+            href="#"
+            data-qa="hover"
+            class="nav-link"
+            >Laptops &amp;Computers</a>
```
Comment (sTorba24) [inline src/index.html:39]:
```suggestion
            <a
              href="#"
              data-qa="hover"
              class="nav-link"
            >Laptops &amp;Computers</a>
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,81 @@
...
+
+body{
+  margin: 0;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
+*{
```
Comment (sTorba24) [inline src/style.css:13]:
```suggestion
* {
```

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,81 @@
-body {
+html{
+  font-family: Roboto, Poppins, sans-serif
+}
+
+body{
```
Comment (sTorba24) [inline src/style.css:5]:
```suggestion
html {
  font-family: Roboto, Poppins, sans-serif
}

body {
```

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,81 @@
...
+  padding: 0;
+}
+
+.logo-images {
+  width: 40px;
+  height: 40px;
   margin: 0;
+  padding: 0;
+  border: 0;
+  display: block;
```
Comment (sTorba24) [inline src/style.css:38]: it's redundant

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,81 @@
...
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+}
```
Comment (sTorba24) [inline src/style.css:30]: can be removed fully

File: src/style.css (hunk 8 above)
Comment (sTorba24) [reply src/style.css:38]: display block you can leave, or set width and height for .logo

**Review by sTorba24 — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 8 above)
Comment (ShaoLei2) [reply src/style.css:38]: Since I completely remove the .logo according to your lower notes, I leave it as it is

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,74 @@
+html {
+  font-family: Roboto, Poppins, sans-serif
```
Comment (l4st1m0za) [inline src/style.css:2]: Missing semicolon at the end of the rule.

File: src/index.html [hunk 11]
```diff
@@ -5,10 +5,53 @@
...
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Smartphones</a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#"
+              data-qa="hover"
+              class="nav-link"
+            >Laptops &amp;Computers</a>
```
Comment (l4st1m0za) [inline src/index.html:39]: There is missing space between '&' and 'Computers'. It should be '& Computers'.

**Review by l4st1m0za — CHANGES_REQUESTED** [review]
Good job but you need to pass all the test. For now you have problems with font and size

File: src/index.html [hunk 12]
```diff
@@ -5,10 +5,53 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+        src="/layout_moyo-header/src/images/logo.png"
+        alt="logo"
+        class="logo-images"
+        >
```
Comment (anastasiiavorobiova) [inline src/index.html:21]:
```suggestion
        <img
          src="/layout_moyo-header/src/images/logo.png"
          alt="logo"
          class="logo-images"
        >
```

File: src/index.html [hunk 13]
```diff
@@ -5,10 +5,53 @@
...
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Smartphones</a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#"
+              data-qa="hover"
+              class="nav-link"
+            >Laptops &amp; Computers</a>
```
Comment (anastasiiavorobiova) [inline src/index.html:39]:
```suggestion
            >
              Laptops &amp; Computers
            </a>
```

File: src/index.html [hunk 14]
```diff
@@ -5,10 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (anastasiiavorobiova) [reply src/index.html:10]: Not fixed

File: src/style.css [hunk 15]
```diff
@@ -1,3 +1,74 @@
...
+  border: 0;
+  display: block;
+}
+
+.nav-list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  gap: 16px;
```
Comment (anastasiiavorobiova) [inline src/style.css:39]:
```suggestion

```
Screenshot 2023-12-05 at 13 21 36

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

File: src/index.html (hunk 13 above)
Comment (ShaoLei2) [reply src/index.html:39]: image
I already told you that it stops working as soon as I do that

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Make sure to deploy your last fixes

**Comment by ShaoLei2** [comment]
мавп

**Pull request closed without merging**